dwave/embedding/chain_breaks.py

```diff
@@ -26,9 +26,58 @@
            'majority_vote',
            'weighted_random',
            'MinimizeEnergy',
+           'break_points',
            ]
 
 
+def break_points(samples, embedding):
+    """Identify breakpoints in each chain.
+
+    Args:
+        samples (array_like):
```

Comment:
I think this should be expanded to [samples like](https://docs.ocean.dwavesys.com/en/stable/docs_dimod/reference/generated/dimod.as_samples.html#dimod.as_samples). You can use `array, labels = dimod.as_samples(samples_like)` to get the numpy array.

The reason I say this is because in Ocean you can get embeddings that look like `{'a': ['b', 'c']}`. The QPU (currently) only uses integer labels for its qubits, but that might change in the future.

Should add a test for this as well.